src/index.js

```diff
@@ -15,9 +16,10 @@ if (process.env.NODE_ENV !== "production") {
 
 const mapConfig = parseUriHash(defaultMapConfig);
 
-new LeafletMap(mapConfig);
+Window.lmap = new LeafletMap(mapConfig);
```

Comment:
see my note about how you can avoid relying on using `this` in `dispatch.call()`

Reply:
@jimboweb not sure if you saw this comment?

Reply:
Yeah, as I said above, leaving the `this` argument undefined did not make the leafletMap `this`.